LICENCE-THIRD-PARTY

```diff
@@ -1,12 +1,48 @@
 The CS Field Guide uses third-party libraries or other resources that may
-be distributed under licenses different than the CS Field Guide license.
+be distributed under licences different than the CS Field Guide licence.
 
-A summary of all third-party licenses are listed below, with a full copy of
-each license available within the 'third-party-licenses' directory.
+A summary of all third-party licences are listed below, with a full copy of
+each licence available within the 'third-party-licences' directory.
 
 If a particular source file of a library is included with the repository, it
 should contain a copyright notice at the top of the file.
 
-In the event that we accidentally failed to list a required license,
+In the event that we accidentally failed to list a required licence,
 please bring it to our attention through the issue tracker here:
 https://github.com/uccser/cs-field-guide/issues/new
+
+==============================================================================
+css3patterns
+------------------------------------------------------------------------------
+https://github.com/LeaVerou/css3patterns
+Copyright 2011 Lea Verou
+licensed under MIT licence.
+third-party-licences/css3patterns.txt
+==============================================================================
+
+==============================================================================
+Detector.js
```

Comment:
This file appears to be licensed incorrectly, as it is built from the 2010-2011 three.js authors (as stated [here](https://github.com/alteredq/three.js/blob/master/LICENSE)), but then modified further by alteredq and mr.doob (see [here](https://github.com/alteredq/three.js/blob/master/examples/js/Detector.js#L2-L3)), then further modified by Alasdair (see [here](https://github.com/uccser/cs-field-guide/blob/b203cabfe2cf657801400b26b10e9742c6f63480/csfieldguide/static/js/third-party/threejs/Detector.js#L8)). Line 27 of this file should reflect that, and it should have it's own file in the `third-party-licences` directory.

However there must be a simpler way to use a file from a 8 year old fork, maybe this would be better (haven't properly researched it): https://stackoverflow.com/questions/11871077/proper-way-to-detect-webgl-support